dev/how-to/get-started/data-migration.md

```diff
@@ -245,7 +245,9 @@ password = ""
 port = 3307
 ```
 
-The `flavor` option should be set to `"mysql"` (the default value, and 5.5 < MySQL versions < 8.0 are supported) if migrating from MySQL Server, Percona Server, Percona XtraDB Cluster, or Amazon Aurora or RDS. If migrating from MariaDB Server or MariaDB (Galera) Cluster, use `flavor = "mariadb"` (only MariaDB versions greater than 10.1.2 are supported).
+- If you migrate data from MySQL Server, Percona Server, Percona XtraDB Cluster, Amazon Aurora or RDS, set the `flavor` option to `"mysql"`, which is the default value. For this default value, MySQL versions between 5.5 (not included) and 8.0 (not included) are supported.
+- If you migrate data from MariaDB Server or MariaDB (Galera) Cluster, set `flavor = "mariadb"`, which only supports MariaDB version later than 10.1.2.
+- Starting from DM 1.0.2, DM automatically generates the `flavor` and `server-id` options. You do not need to manually configure these options in normal situations.
```

Comment:
```suggestion
- Starting with DM 1.0.2, DM automatically generates values for the `flavor` and `server-id` options. You do not need to manually configure these options in normal situations.
```
Please confirm with the author on "which" DM automatically generates - "the values of the options" or "the options + their values". @TomShawn @csuzhangxc

Reply:
"the values of the options"